apidoc/Titanium/UI/TextArea.yml

```diff
@@ -187,6 +187,26 @@ properties:
     default: No hint text.
     platforms: [android, blackberry]
     
+  - name: hintTextColor
+    summary: Hint text color.
```

Comment:
hmm summary seems incomplete, maybe "Color of hint text that displays when field is empty", similar to hintText docs.